dev-python/pymupdf/pymupdf-1.19.6.recipe

```diff
@@ -0,0 +1,78 @@
+SUMMARY="Python bindings for MuPDF's rendering library"
+DESCRIPTION="PyMuPDF is a Python binding with support for MuPDF, a lightweight PDF, XPS, and \
+E-book viewer, renderer, and toolkit, which is maintained and developed by Artifex Software, Inc.
+MuPDF can access files in PDF, XPS, OpenXPS, CBZ, EPUB and FB2 (e-books) formats, and it is known \
+for its top performance and high rendering quality.
+With PyMuPDF you can access files with extensions like \".pdf\", \".xps\", \".oxps\", \".cbz\", \
+\".fb2\" or \".epub\". In addition, about 10 popular image formats can also be handled like \
+documents: \".png\", \".jpg\", \".bmp\", \".tiff\", etc..
+"
```

Comment:
Move closing double-qoute to the end of the previous line:
```diff
-documents: \".png\", \".jpg\", \".bmp\", \".tiff\", etc..
-"
+documents: \".png\", \".jpg\", \".bmp\", \".tiff\", etc.."
```

Reply:
Done